[DOC] Some minor typos fixed in tutorials (#2917)
seqan/seqan3: doc/tutorial/ranges/index.md

  * sorting only all elements after the fifth one: `std::ranges::sort(std::views::drop(v, 5))`
  * sorting in reverse order: `std::ranges::sort(std::views::reverse(v))`

> We will discuss later what `std::views::reverse(v)` does, for now, it is enough to understand that it returns something

Reviewer: ```suggestion
We will discuss later what `std::views::reverse(v)` does. For now, it is enough to understand that it returns something
```

That's because `std::views::reverse` is not the view itself, it's an *adaptor* that takes the underlying range
(in our case the vector) and returns a view object over the vector.
The exact type of this view is hidden behind the `auto` statement.

> This has the advantage that we don't need to worry about the template arguments of the view type, but more importantly
> the adaptor has an additional feature: it can be *chained* with other adaptors!

Reviewer: ```suggestion
This has the advantage that we don't need to worry about the template arguments of the view type. 
The adaptor has another, very useful feature: it can be *chained* with other adaptors!
```

I had to read the sentence a few times to notice that the feature of chaining has nothing to do with the explanation of the advantage of `auto` :)

With few exceptions, views don't model more/stronger range concepts than their underlying range (other than
std::ranges::view) and they try to preserve as much of the underlying range's concepts as possible.
For instance the view returned by `std::views::reverse` models std::ranges::random_access_range (and weaker concepts)

> if the underlying range also models the respective concept.

Reviewer 2: I don't know if you are aware of `iff` (https://en.wikipedia.org/wiki/If_and_only_if)
but I think you are right that in this case `iff` does not apply because the former statement is always true regardless of the latter.

We provide overview tables for all our view adaptors that document which concepts are modelled by the views they return.

> ## Views in the standard library and SeqAn

Reviewer 3: The in seems correct to me
